scripts/convert_vcf_format.R

```diff
@@ -10,7 +10,7 @@ options(scipen = 999)
 # Create a parser object
 if(!interactive()){
   argp = arg_parser("convert VCF format")
-  argp = add_argument(argp, "--reference", default="", help="Reference genome to use. Must be a valid installed BSgenome package")
+  argp = add_argument(argp, "--reference", default="", help="Reference fasta genome file to use.")
```

Comment:
```suggestion
  argp = add_argument(argp, "--reference", default="", help="Reference genome fasta file to use.")
```

Reply:
regarding you question above, so in the manual it says that it should be indexed

"Use FaFile() to create a reference to an indexed fasta file. The reference remains open across calls
to methods, avoiding costly index re-loading."

I tried to run the Drosophila file they gave us, with and without that fai, it took the same time. So maybe it is still fine, or the file is too small to judge, but anyway it does not fall